experiment/autobumper/bumper/bumper.go

```diff
@@ -116,6 +116,8 @@ type Options struct {
 	RemoteName string `yaml:"remoteName"`
 	// The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
 	HeadBranchName string `yaml:"headBranchName"`
+	// Optional list of labels to add to the bump PR
+	Labels []string `yaml:"labels"`
```

Comment:
We probably have discussed about this before 😅 , is `omitempty` needed?

Reply:
I don't believe it is needed. The default value is []. I tested running without labels in the config and it worked as intended. I think omitempty helps when going from a golang struct -> json, but it doesn't seem to do anything when going from json -> struct.